### ISSUE TITLE
Extend addSQLiteSuffix and deleteOldDatabases wih a name regex or list

### ISSUE THREAD

**Issue by mobilemarines** [capacitor-community/sqlite#182]
Hey,

When I try to migrate our old Cordova SQLite database with addSQLiteSuffix at other databases in that folder will be migrated, too. There are other databases in that directory because other plugins need to create their database in there too and are not 
 also not configurable here. And another problem is that if we successfully migrate the correct db and cleanup with deleteOldDatabases() afterwards also the wrong database which shouldn't be migrated would've been deleted, too.

So a solution that would fit our needs would be something like an expansion of the both mentioned functions with a name regex or a list of names, that should be migrated/deleted in the given directory.

Let me know if you need more information about it and thank you in advance.

Regards,
mobilemarines

**Comment by jepiqueau** [comment]
@mobilemarines Thank for using the plugin. Interesting i will look at this , if there is no db list given all db will be copied/deleted if there is a list provided only those part of the list will be copied/deleted. Is that will fit your needs?

**Comment by mobilemarines** [comment]
Hey @jepiqueau thanks a lot for your quick response, sorry that it  took a while to answer, we had a long weekend here ;-) 
The solution would fit our needs very well, a simple list of names, of which databases to delete/migrate. The problem with the migration is also that you can only get a list of databases that are already migrated or created with this plugin by calling getDatabaseList(). So I did't look how that function works internally but we're using a workaround to get a list of files in the directory to see if the migration is necessary or not. So maybe this would be another change request. 

Hope I didn't intimidate you with all those requests ;-)

Have a great week,
mobilemarines

**Comment by jepiqueau** [comment]
@mobilemarines What i am working on is `getMigratableDbList(folderpath: string)` is this will fit your need

**Comment by mobilemarines** [comment]
@jepiqueau I tested the release with Android and iOS in our project and it worked flawlessly. The only thing I recognized is that 
 `addSQLiteSuffix()`changes the path of the library, at least i see this behaviour on iOS, we have the path `Library/LocalDatabase` and after the migration the database is in `Documents/` so i had to handle that in a special case. Maybe this is misleading and could be documented.
